### PR TITLE
Oss fuzz fixes, another batch

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/core/load-fragment.c [hunk 1]
```diff
@@ -226,6 +226,10 @@ int config_parse_unit_path_printf(
         assert(rvalue);
         assert(u);
 
+        /* Let's not bother with anything that is too long */
+        if (strlen(rvalue) >= PATH_MAX)
+                return -ENAMETOOLONG;
```
Comment (poettering) [inline src/core/load-fragment.c]: should also invoke log_syntax() I figure

File: src/basic/calendarspec.c [hunk 2]
```diff
@@ -197,6 +197,10 @@ _pure_ static bool chain_valid(CalendarComponent *c, int from, int to, bool end_
         if (c->start < from || c->start > to)
                 return false;
 
+        /* Avoid overly large values that could cause overflow */
+        if (c->repeat > to - from)
+                return false;
```
Comment (poettering) [inline src/basic/calendarspec.c:204]: hmm, if we calc "to - from" now, we should maybe also add an explicit assert before that assures that "to >= from" is. I mean, it is in all calls we make to this function, but we should denote that in code I am sure, just to be safe.

**Comment by poettering** [comment]
looks good. just two nitpicks.

**Comment by keszybz** [comment]
Thanks for the review. Updated.

**Pull request merged**